dev_course/dl2/02b_initializing.ipynb

```diff
@@ -6,7 +6,8 @@
    "metadata": {},
```

Comment:
<p>same as above. but if you keep it as the original then your proof doesn't match.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_docs/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>